libr/core/cmd_print.c

```diff
@@ -1839,6 +1839,15 @@ static void disasm_strings(RCore *core, const char *input, RAnalFunction *fcn) {
 		}
 		if (addr != UT64_MAX) {
 			const char *str = NULL;
+			char *comment = r_core_anal_get_comments (core, addr);
+			bool show_comments = r_config_get_i (core->config, "asm.comments");
+			RConsPalette *pal = &core->cons->pal;
+			if (comment && show_comments) {
```

Comment:
only get the comment if asm.comments is enabled. also for performacne reasons it will better to move r_config_get_i from this loop and place it at the begining of the function to only check it once